README.md

```diff
@@ -63,13 +63,22 @@ Use the following environment variables to configure your container:
 |`INSP_ADMIN_NICK`        |`MI5`                           |Nick showed by the `/admin` command           |
 |`INSP_ADMIN_EMAIL`       |`jonny.english@example.com`     |E-mail shown by the `/admin` command          |
 |`INSP_ENABLE_DNSBL`      |`yes`                           |Set to `no` to disable DNSBLs                 |
+|`INSP_CONNECT_PASSWORD`  |`` (empty)                      |Enables a server password, empty to disable   |
```

Comment:
Could you rename this to `INSP_CONNECT_PASSWORD_HASH` as we already do with the oper password? Because this can easily cause confusion.

Reply:
I think there will be confusion with using the word `hash` in the password variable as it can be in plaintext, un-hashed. I named the environment variables similar to the config items in the `<connect ..>` block.

With these two configuration options we can mix them to create 3 scenarios for a connection block/users(s):

hash | password | auth | config-file
------------ | ------------- | ------------- | -------------
empty | empty | no authentication | empty
empty | [password] | requires server password | [password] aka plaintext
[hash-algo] | [password] | requires server password | hash([password])